tompscanlan/packerd
# Large update

- re-gen using latest swager libs
- expose logging via command line args
- add disk free check to health
- get a docker clean worker going
- add artifact recording
- clean docker images after build finishes so we don't run out of space
- don't docker clean while kitchen is running
- don't run docker clean while packer is running
- add a container def for pushing to ECS
- move to port 81, nginx takes 80 interally
- change httpx_proxy to httpx_proxy_server, so we can also pass in an official proxy to the env
- try using general proxy, but allow job to pass in better proxy
- filter out proxy settings from env while running commands
- update readme
